content/sales.rst

```diff
@@ -0,0 +1,196 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describe operational Sales details.
+
+The main contact is Tim Adams, the main email group is at sales@proatria.com.
+
+Sales:
+
+* Provides the main software information to prospective customers, forwards
```

Comment:
I hope that we don't have a separate sales team, and the support team will also handle sales.

again, the styleguide should document exactly what we do.

We don't have a sales team... and support is doing a lot of sales.

Yes. Tim can be seen as a sales member... but in fact he is doing support.